weather_dl/download_pipeline/parsers.py

```diff
@@ -30,6 +31,38 @@
 Config = t.Dict[str, t.Dict[str, Values]]
 
 
+@dataclasses.dataclass()
+class Config:
+    """Contains pipeline parameters
+
+    Attributes:
+        client: Name of the Weather-API-client. Supported clients are mentioned in the 'CLIENTS' variable.
+        dataset (optional): Name of the target dataset. Allowed options are dictated by the client.
+        partition_keys (optional): Choose the keys from the selection section to partition the data request.
+                                   This will compute a cartesian cross product of the selected keys
+                                   and assign each as their own download.
+        target_path: Download artifact filename template. Can make use of Python's standard string formatting.
+                     It can contain format symbols to be replaced by partition keys;
+                     if this is used, the total number of format symbols must match the number of partition keys.
```

Comment:
The copy is good here. However, the indentation is a bit unconventional. The the following style guide for examples on proper indentation: 

https://google.github.io/styleguide/pyguide.html#s3.8-comments-and-docstrings